### DISCUSSION TITLE
[Suggestion] Change 'Color' to 'SpectreColor'

### DISCUSSION THREAD

**Discussion by jeffward01** [spectreconsole/spectre.console#770] [General]
Hello!

I suggest that 'Color' from `Spectre.Console.Color` is change to `Spectre.Console.SpectreColor`.

The reason for this change is that `Color` is incredibly popular, and has conflicting names in just about every single Console UI library.

I find myself every time when working with `Spectre.Console` that I must perform a using statement declaration of:

````csharp
using Spectre.Console;
using SpectreColor = Spectre.Console.Color;   
````

# Suggestion

I suggest: `Spectre.Console.Color` is changed to `Spectre.Console.SpectreColor`.

I know its a breaking change, I can see this being a 'blocker', due to this.  

It will be a simple 'breaking change'.

**Comment by patriksvensson** [comment]
@jeffward01 I have personally never encountered that problem, and I don't see us changing to `SpectreColor`, to be honest.
In C# 10, you can add a global using once in your project, which should solve your problem:

```csharp
global using SpectreColor = Spectre.Console.Color;
```

**Reply by jeffward01** [reply]
I agree, that:

````csharp
global using SpectreColor = Spectre.Console.Color;
````

is a good solution, however I find myself doing this on all my projects.  

# Reasons
Also I see that ([Spectre.Console.ImageSharp](https://github.com/spectreconsole/spectre.console/packages/517350?version=0.42.1-preview.0.18)) is a popular extension for Spectre.Console, a library which I use often.

This library ([Spectre.Console.ImageSharp](https://github.com/spectreconsole/spectre.console/packages/517350?version=0.42.1-preview.0.18)) is a 'packaged' version of [ImageSharp](https://github.com/SixLabors/ImageSharp).  The library [ImageSharp ](https://github.com/SixLabors/ImageSharp) (among many other libraries which are commonly used in console apps) use the class `Color`.  [Here is how](https://github.com/SixLabors/ImageSharp/blob/e54219acec10d2d2dc247ed5fd4230204a89c07b/tests/ImageSharp.Tests/Color/ReferencePalette.cs) `Color` is used in [ImageSharp ](https://github.com/SixLabors/ImageSharp) . 

It is common that Visual Studio references the namespace `SixLabors.ImageSharp` instead of `Spectre.Console.Colors` because of the overlapping namespace names.

# Summary
While I agree that the **global** using is a good solution, I feel that because many libraries use the `Color` class, including libraries which are integrated into extension libraries of `Spectre.Console`, it would be nice to have a unique naming concept of `Color` to `SpectreColor`.

I agree that this is more of a 'Convenience' and 'Cosmetic' fix and discussion, but I think it would make sense. 

## Proposal
However there would be breaking changes of course due to the class name change of `Color` to `SpectreColor`. So perhaps wait until the next major version release.

I love `Spectre.Console` and I just wanted to share my opinion on this.

Thanks!
